Labelbox/labelbox-python
# [PTDT-1113] Add delete_unused_ontology method

- calls a new REST endpoint, what should help with SDK4.0 migration
- delete_unused_ontology can be used to delete an ontology

### How to test

If https://github.com/Labelbox/intelligence/pull/13388 is not merged, git checkout lgluszek/PTDT-1113 in `intelligence`

Usage:

```python
from labelbox import Client

client = Client(
    api_key="...",
    rest_endpoint="http://localhost:3000/api/api/v1")

res = client.delete_unused_ontology("...")
```

There should be an error when user tries to delete an ontology that is connected to at least 1 project
